client/state/data-layer/http/README.md

```diff
@@ -0,0 +1,52 @@
+# Raw HTTP API Layer
+
+[`wpcom-http`](../wpcom-http) issues requests to the WordPress.com API. However sometimes you will want to issue a request to another API, maybe even on a third-party host. This raw HTTP layer will allow you to do just that.
+
+It follows the same API than `wpcom-http`, the only difference from the user's perspective being the action dispatched.
```

Comment:
quick nitpick: "It follows the same API _as_ the `wpcom-http`" or "It shares the same API as the `wpcom-http` layer"